From bdf5fe3ab3d74b4dc1547ffd04324b79b141a7d6 Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Fri, 18 Sep 2020 11:50:11 -0700
Subject: [PATCH 2/2] HACK: Use WORKSPACE instead of HOME

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 src/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 0ef6065..e628fad 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -16,7 +16,7 @@ MINIFE_MATRIX_TYPE = -DMINIFE_CSR_MATRIX
 
 # BG/Q
 #CFLAGS = -DIBM_HPM -qlist -qsource  -g -O3 -qsmp=omp
-CFLAGS = -g -O3 -fopenmp -I$(HOME)/build/geopm/include
+CFLAGS = -g -O3 -fopenmp -I$(WORKSPACE)/build/geopm/include
 CXXFLAGS = $(CFLAGS)
 
 # To enable memory high water mark reporting uncomment the
@@ -25,7 +25,7 @@ CPPFLAGS = -I. -I../utils -I../fem $(MINIFE_TYPES) \
 	$(MINIFE_MATRIX_TYPE) \
 	-DHAVE_MPI -DMPICH_IGNORE_CXX_SEEK #-DMINIFE_REPORT_RUSAGE
 
-LDFLAGS=$(CFLAGS) -L$(HOME)/build/geopm/lib
+LDFLAGS=$(CFLAGS) -L$(WORKSPACE)/build/geopm/lib
 LIBS=-lgeopm
 
 # GNU CXX=mpicxx
-- 
2.8.3

